### ISSUE TITLE
图像代理模型的优化

### ISSUE THREAD

**Issue by tjuxiaoli** [tswang0116/EQB2A#2]
你好，请问下面的图像代理模型的构造与句子中， for p in self.parameters():  p.requires_grad = False不是会把代理模型参数都冻结了吗，那后面训练代理模型的时候，这个优化器optimizer_SIM = torch.optim.Adam(filter(lambda p: p.requires_grad, self.SIM.parameters()), lr=self.args.kilr, betas=(0.5, 0.999))不就是没用了吗，请问是需要自己训练的时候改为true吗，还是图像代理模型就是不用再优化。
class SurrogateImageModel(nn.Module):
    def __init__(self, bit):
        super(SurrogateImageModel, self).__init__()
        self.bit = bit
        self.model = torchvision.models.resnet50(pretrained=True)
        self.model.fc = nn.Sequential()
        for p in self.parameters():
            p.requires_grad = False
        self.surrogate_model = nn.Sequential(
            nn.Linear(2048, 1024),
            nn.ReLU(inplace=True),
            nn.Dropout(0.3),
            nn.Linear(1024, self.bit),
            nn.Tanh())
